### PR TITLE
update README links to point to vox namespace

### PR DESCRIPTION
**Pull request by 3flex**
Updated internal links to github and Travis

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 ## INSTALLING OR UPGRADING
 ** Please note **: This module is currently undergoing some structural
-maintenance. Please take a look at [https://github.com/jfryman/puppet-nginx/blob/master/docs/hiera.md](https://github.com/jfryman/puppet-nginx/blob/master/docs/hiera.md)
+maintenance. Please take a look at [https://github.com/voxpupuli/puppet-nginx/blob/master/docs/hiera.md](https://github.com/voxpupuli/puppet-nginx/blob/master/docs/hiera.md)
 before upgrading or installing Version 0.1.0 or greater.
 
 [![Puppet
 Forge](http://img.shields.io/puppetforge/v/jfryman/nginx.svg)](https://forge.puppetlabs.com/jfryman/nginx)
```
Comment (igalic) [inline README.markdown]: this one still is pointing jfryman

Comment (3flex) [inline README.markdown]: I'm aware, but there's no module on Forge under voxpupuli yet. https://forge.puppet.com/puppet/nginx gets you a 404. Happy to update to change this link also? I'm assuming the new module is uploaded soon enough.

Comment (igalic) [inline README.markdown]: i would definitely suggest we do a release under the new namespace, asap.

**Comment by bastelfreak** [comment]
sorry, I missed this PR. We just released in https://github.com/voxpupuli/puppet-nginx/pull/868

**Pull request closed without merging**